.gitban/templates/chore-triage.md

```
@@ -0,0 +1,233 @@
+| **REJECT** | [e.g., 1] | [e.g., 10%] | [e.g., Team decision documented] |
+| **JUNK** | [e.g., 0] | [e.g., 0%] | [e.g., No test/spam cards in this batch] |
+| **Total** | [e.g., 10] | 100% | |
+
+### Archive Actions
+
+| Archive Sprint Name | Card IDs Archived | Dispositions |
+| :--- | :--- | :--- |
+| [e.g., 2025-Q4-triage] | [e.g., F0123, B0456, C0789] | [e.g., DONE (1), STALE (1), DUPE (1)] |
+
+**Archive Command Used:**
+```python
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)